package/MDAnalysis/topology/PDBParser.py

```diff
@@ -178,6 +178,10 @@ class PDBParser(TopologyReaderBase):
      - bonds
      - formalcharges
 
+    Note that `PDBParser` accepts an optional keyword argument
```

Comment:
Thanks for adding this.

Our docs for the parsers are really bad.... no kwargs are actually documented.